bip-raw-as-subscript.md

```diff
@@ -0,0 +1,101 @@
+<pre>
+  BIP: ???
+  Layer: Applications
+  Title: raw() as subscript in Output Script Descriptors
+  Author: Matias Furszyfer <mfurszy@protonmail.com>
+  Comments-Summary: No comments yet.
+  Comments-URI: https://github.com/bitcoin/bips/wiki/Comments:BIP-???
+  Status: Draft
+  Type: Standards Track
+  Created: ?
+  License: BSD-2-Clause
+</pre>
+
+==Abstract==
+
+This document specifies `raw()` as subscript for output script descriptors.
+`raw()` encapsulates a raw hex script. This BIP allows `raw()` to be used in
+the context of other descriptors.
+
+==Copyright==
+
+This BIP is licensed under the BSD 2-clause license.
+
+==Motivation==
+
+Allowing arbitrary hex data to be wrapped in `sh()`, `wsh()`, or even within the `TREE`
+argument of a `tr(KEY, TREE)` descriptor enables the representation of currently
+inexpressible information in the descriptors' language.
+
+Specifically, the absence of this feature limits the representation of non-standard redeem
+and witness scripts. This occurs because they can currently only be represented as top-level
+`raw(HEX)` descriptors, which retain only the output script information and lack the ability
+to preserve the actual script.
+
+Additionally, as noted [here](https://github.com/bitcoin/bitcoin/issues/24114#issuecomment-1127978154),
+there are other useful scenarios for this feature. For example, it allows representing
+in a descriptor that we lack complete knowledge of all solvability paths but can still
+solve the output. This includes cases like a taproot tree where we know only one of its
```

Comment:
This is a bit unclear for me, can you clarify what do you mean by can still solve the output?